test/src/test/resources/transport/grpcSaasConfig.properties

```diff
@@ -0,0 +1,18 @@
+# Transport Plugin Configuration
+transport_adapter_JAR_path=../infrastructure/target/infrastructure-0.4.0-SNAPSHOT.jar
+transport_adapter_server_class_name=org.corfudb.infrastructure.logreplication.transport.sample.GRPCLogReplicationServerChannelAdapter
+transport_adapter_client_class_name=org.corfudb.infrastructure.logreplication.transport.sample.GRPCLogReplicationClientChannelAdapter
+
+# Stream Fetcher Plugin Configuration
+stream_fetcher_plugin_JAR_path=../infrastructure/target/infrastructure-0.3.1-SNAPSHOT.jar
+stream_fetcher_plugin_class_name=org.corfudb.infrastructure.logreplication.infrastructure.plugins.DefaultLogReplicationConfigAdapter
+
+# Topology Manager Plugin Configuration
+topology_manager_adapter_JAR_path=../infrastructure/target/infrastructure-0.4.0-SNAPSHOT.jar
+topology_manager_adapter_class_name=org.corfudb.infrastructure.logreplication.infrastructure.plugins.DefaultClusterManager
+
+# Snapshot Sync Configuration (Plugin)
+snapshot_sync_plugin_JAR_path=../infrastructure/target/infrastructure-0.3.1-SNAPSHOT.jar
+snapshot_sync_plugin_class_name=org.corfudb.infrastructure.logreplication.infrastructure.plugins.DefaultSnapshotSyncPlugin
+
+saas_endpoint=corfu:9000
```

Comment:
this means that a separate properties file will be used for on-prem and saas deployments, is that right?

Reply:
Actually no..this was a miss on my end. Have changed the a bit now
LR will ask cluster manager if the current deployment is Saas. If yes, then endpoint is computed [here](https://github.com/CorfuDB/CorfuDB/blob/saasEndpoint/infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/Utils/CorfuSaasEndpointProvider.java#L29), otherwise its an empty optional.